networking/eth2/src/integration-test/java/tech/pegasys/artemis/networking/eth2/GoodbyeIntegrationTest.java

```diff
@@ -48,7 +46,5 @@ public void shouldCloseConnectionAfterGoodbyeReceived() throws Exception {
     waitFor(peer1.sendGoodbye(GoodbyeMessage.REASON_CLIENT_SHUT_DOWN));
     Waiter.waitFor(() -> assertThat(peer1.isConnected()).isFalse());
     Waiter.waitFor(() -> assertThat(peer2.isConnected()).isFalse());
-    assertThat(network1.getPeerCount()).isZero();
-    assertThat(network2.getPeerCount()).isZero();
```

Comment:
Because it reconnects a lot faster now so while the explicit peer instance is guaranteed to be disconnected, it may have reconnected again before we get to the peerCount is zero check.